andela/colossus-backend
# #168378880 Edit posted comments on a request

**What does this PR do?**
-this gives a user the ability to edit their comments on a request.

**Description of task to be completed**
-implement the logic for editing a prevoiusly posted comment
-ensure the route PATCH /api/v1/requests/requestId/comment/commentId is working perfectly

**How should this be manually tested?**
once this repo is integrated
-on postman sign in a user who previously  commented on a request, ensure you know the exact comment && request ID  then 
make a PATCH request to /api/v1/requests/requestId/comment/commentId with a simple and valid
comment object like this => { commentBody: "I want to patch"  }

**Any background contexts?**
N/A

**Relevant PT story**
#168378880
